File: Iniciante/URI 1963.cpp
#include <iostream>

int main() { 
    float A, B, aumento;

    scanf("%f %f", &A, &B);
	aumento = (B * 100.0 / A) - 100.0;
    printf("%.2f%%\n", aumento);

    return 0;
}

File: Iniciante/URI 1963.go
package main
 
import (
    "fmt"
)

func main() {
    var A, B, aumento float64

    fmt.Scanf("%f", &A)
    fmt.Scanf("%f", &B)
    aumento = (B * 100.0 / A) - 100.0
    fmt.Printf("%.2f%%\n", aumento)
}